FAQ.md

```diff
@@ -108,6 +110,13 @@ trying to figure out why `dep` is doing what it does, understanding that its
 basic action is to attempt versions in this order should help you to reason
 about what's going on.
 
+## What external tools are supported?
+During `dep init` configuration from other dependency managers are detected
+and imported, unless `-skip-tools` is specified.
+The following tools are supported: glide, and godep. See 
+[#186](https://github.com/golang/dep/issues/186#issuecomment-306363441) for
+how to add support for another tool.
```

Comment:
* It should be ... configuration from other dependency managers **is** ...
* Empty lines could help improving readability. The whole answer is squashed into one line.
* Should we use `glide` and `godep` instead?

```
During `dep init` configuration from other dependency managers is detected
and imported, unless `-skip-tools` is specified.

The following tools are supported: `glide` and `godep`.

See  [#186](https://github.com/golang/dep/issues/186#issuecomment-306363441) for
how to add support for another tool.
```

Also, I think this question should be the second one (right after "Does `dep` replace `go get`?")